problem-23/README.md

```diff
@@ -0,0 +1,11 @@
+# Task 23: Reset form when submitted (lv 1)
+
```

Comment:
Included in the task is for the user to create a comment with the services they offer.

The only task now is to clear the input values. I think that was your initial thought.

Reply:
Resolved - changed the read me to make it more clear. The task is to reset the form input values when you submit the form. Let me know if it is not clear.